rskj-core/src/main/java/co/rsk/net/discovery/PeerExplorer.java

```diff
@@ -204,6 +218,13 @@ private void handlePingMessage(InetSocketAddress address, PingPeerMessage messag
     private void handlePong(InetSocketAddress pongAddress, PongPeerMessage message) {
         PeerDiscoveryRequest request = this.pendingPingRequests.get(message.getMessageId());
 
+        logger.debug("handlePong - Handling pong message with " +
+                        "pongAddress hostName: [{}], " +
+                        "pongAddress port: [{}], " +
+                        "messageId: [{}], " +
+                        "request: [{}]"
+                , pongAddress.getHostName(), pongAddress.getPort(), message.getMessageId(), request);
+
         if (request != null && request.validateMessageResponse(pongAddress, message)) {
```

Comment:
I guess we should print a WARN here if request is null or invalid

Reply:
Done